oar/cli/cmd_group.py

```diff
@@ -38,7 +38,7 @@ def print_version(ctx, param, value):
     expose_value=False,
     is_eager=True,
 )
-@click.option("-r", "--release", help="z-stream releaes version")
+@click.option("-r", "--release", help="z-stream release version", required=True)
```

Comment:
required flag is removed due to enhancement [OCPQE-25622](https://issues.redhat.com/browse/OCPQE-25622). validation will be done implicitly